scripts/apps/authoring/authoring/controllers/ChangeImageController.ts

```diff
@@ -13,23 +13,49 @@ import {appConfig} from 'appConfig';
  * @requires lodash
  * @requires api
  * @requires $rootScope
+ * @required content
  *
  * @description Controller is responsible for cropping pictures and setting Point of Interest for an image.
  */
 
-export function validateMediaFieldsThrows(validator, metadata) {
-    for (let key in validator) {
-        const value = metadata[key];
+export function validateMediaFieldsThrows(validator, metadata, schema) {
+    const ERROR = gettext('Required field(s) missing');
+
+    Object.keys(validator).forEach((key) => {
+        if (!validator[key].required) {
+            return;
+        }
+
+        const specs = schema[key] || null;
```

Comment:
What does `specs` stand for? Maybe `fieldOptions` or `schemaItem` would be a better name?